### PR TITLE
chore(Report): Bump Viz Versions

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@dcamper - bumped all versions and cleaned up some of the loader code.

**Comment by dcamper** [comment]
@GordonSmith Why is .vscode now included?

**Comment by GordonSmith** [comment]
Its only used for Dev + Debug work (I had deleted the DataPatterns folder on my machine and had to recreate it, don't want to do it again)

**Comment by GordonSmith** [comment]
The only files from a runtime perspective are the ones in:  https://github.com/hpcc-systems/DataPatterns/blob/master/Profile.manifest

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Chrome",
+            "request": "launch",
+            "type": "pwa-chrome",
+            "url": "file:///C:/Users/gordon/git/DataPatterns/report/res/index.html?W20210317-085811",
```
Comment (GordonSmith) [inline .vscode/launch.json:11]: Replace `C:/Users/gordon/git/DataPatterns` with `${workspaceFolder}`

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (GordonSmith) [inline .vscode/launch.json:4]: Remove comments

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            "webRoot": "${workspaceFolder}"
+        },
+        {
+            "name": "play.hpccsystems.com",
+            "type": "ecl",
+            "request": "launch",
+            "protocol": "https",
+            "serverAddress": "play.hpccsystems.com",
+            "port": 18010,
+            "targetCluster": "hthor",
```
Comment (GordonSmith) [inline .vscode/launch.json:21]: Change to "thor"

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            "rejectUnauthorized": false,
+            "resultLimit": 100,
+            "timeoutSecs": 60,
+            "user": "vscode_user",
```
Comment (GordonSmith) [inline .vscode/launch.json:25]: Change to `datapatterns_developer`

File: report/runTest.ecl [hunk 5]
```diff
@@ -1,6 +1,6 @@
 IMPORT ^ as DataPatterns;
 
-filePath := '~stock_data::cleaned_data';
+filePath := '~class::pfb::out::vehiclereexpanded';
```
Comment (GordonSmith) [inline report/runTest.ecl:3]: @dcamper - you may have a better test file than this?

**Comment by dcamper** [comment]
Well, in my dev environment DataPatterns is a subfolder and the parent folder is what is opened in VS Code.  Wouldn't this embedded .vscode directory override some values found in the parent that kind of configuration?  I may be misremembering a problem I had at one time....

**Comment by GordonSmith** [comment]
@jeclrsg - had an elegant suggestion for this case (rename to launch.json.sample) I will do that.

@jeclrsg please review

**Comment by dcamper** [comment]
@GordonSmith The latest changes to .vscode make more sense.  Thanks.

**Comment by GordonSmith** [comment]
@dcamper - tidied up the `.vscode` folder as per above comments.

**Review by dcamper — APPROVED** [review]

**Comment by dcamper** [comment]
@GordonSmith Please target candidate-1.8.2 instead of master.

**Pull request merged**